src/frameworks/next/interfaces.ts

```diff
@@ -0,0 +1,31 @@
+import type { Header, Rewrite, Redirect } from "next/dist/lib/load-custom-routes";
+
```

Comment:
Unfortunately `Redirect` is not extendable because its members are not statically known, so this is not possible:
```
export interface RoutesManifestRedirect extends Redirect {
  regex: string;
}
```

Reply:
:(